js/spbc-settings_tab--traffic_control.min.js.map

```diff
@@ -1 +1 @@
callback,\n\t\t\t};\n\t\t\tspbc_sendAJAXRequest( data, params );\n\t\t}else{\n\t\t\t// Hide \"More logs\" button, show \"Control Panel\" button\n\t\t\tjQuery(this).hide();\n\t\t\tjQuery(this).siblings('.spbc__show_more_logs').css('display', 'inline-block');\n\t\t}\n\t});\t\n}\n\njQuery(document).ready(function(){\n\t\n\tspbc_tbl__row_actions__listen(); // Table. Row actions handler\n\tspbc_tbl__pagination__listen();  // Table. Pagination handler\n\tspbc_tbl__sort__listen();        // Table. Sort handler\n\t\n\t// Start to hide long values in a table\n\tspbcStartShowHide();\n\t\n\t// FIREWALL LOGS EVENTS\n\t\n\t// Handler for show more FIREWALL LOGS\n\tspbc_tc__show_more__listen();\n\t\n\t// Timer for FireWall logs\n\tvar spbcFireWallLogsUpdateTimer = setTimeout(function spbc_heartbeat(){\t\t\n\t\t// Do refresh only if Traffic Control is enabled and tab is active\n\t\tif( +spbcSettingsFWLogs.moderate && jQuery('.spbc_tab_nav-traffic_control').hasClass('spbc_tab_nav--active')){\n\t\t\tvar data = {\n\t\t\t\taction: 'spbc_show_more_security_firewall_logs',\n\t\t\t\targs: eval('args_'+jQuery('#spbc_tbl__traffic_control_logs').attr('id')),\n\t\t\t\tfull_refresh: true,\n\t\t\t};\n\t\t\tvar params = {\n\t\t\t\twrapper: jQuery('#spbc_tbl__traffic_control_logs'),\n\t\t\t\tcallback: spbc_tc__show_more__callback,\n\t\t\t\tnotJson: true,\n\t\t\t};\n\t\t\tspbc_sendAJAXRequest( data, params );\n\t\t}\n\t\tsetTimeout(spbc_heartbeat, 60000);\n\t}, 60000);\t\n});"],"names":["spbc_tc__show_more__callback","result","data","params","full_refresh","jQuery","wrapper","html","spbc_tbl__row_actions__listen","spbc_tbl__pagination__listen","spbc_tbl__sort__listen","spbcStartShowHide","size","button","hide","siblings","css","spbc_tc__show_more__listen","on","spbcSettingsFWLogs","clicks","action","amount","args","eval","attr","this","spinner","nextElementSibling","find","callback","spbc_sendAJAXRequest","document","ready","spbcFireWallLogsUpdateTimer","setTimeout","spbc_heartbeat","moderate","hasClass","notJson"],"mappings":"AACA,SAASA,6BAA6BC,EAAQC,EAAMC,GAChDD,EAAKE,cACPC,OAAOF,EAAOG,SAASC,KAAKN,GAC5BO,gCACAC,+BACAC,yBACAC,qBACsB,EAAdV,EAAOW,MACfP,OAAOF,EAAOG,SAASC,KAAKN,EAAOM,MACnCC,gCACAG,sBAGAN,OAAOF,EAAOU,QAAQC,OACtBT,OAAOF,EAAOU,QAAQE,SAAS,yBAAyBC,IAAI,UAAW,iBAIzE,SAASC,6BACRZ,OAAO,kCAAkCa,GAAG,QAAS,WACpD,IAEKhB,KAKAC,OAPFgB,mBAAmBC,OAAS,GAC9BD,mBAAmBC,SACflB,KAAO,CACVmB,OAAQ,wCACRC,OAAQH,mBAAmBG,SAAWH,mBAAmBC,OAAO,GAChEG,KAAMC,KAAK,QAAQnB,OAAO,mCAAmCoB,KAAK,QAE/DtB,OAAS,CACZU,OAAQa,KACRC,QAASD,KAAKE,mBACdtB,QAASD,OAAO,mCAAmCwB,KAAK,SACxDC,SAAU9B,8BAEX+B,qBAAsB7B,KAAMC,UAG5BE,OAAOqB,MAAMZ,OACbT,OAAOqB,MAAMX,SAAS,yBAAyBC,IAAI,UAAW,mBAKjEX,OAAO2B,UAAUC,MAAM,WAEtBzB,gCACAC,+BACAC,yBAGAC,oBAKAM,6BAGA,IAAIiB,4BAA8BC,WAAW,SAASC,iBAErD,IACKlC,KAKAC,QANAgB,mBAAmBkB,UAAYhC,OAAO,iCAAiCiC,SAAS,0BAChFpC,KAAO,CACVmB,OAAQ,wCACRE,KAAMC,KAAK,QAAQnB,OAAO,mCAAmCoB,KAAK,OAClErB,cAAc,GAEXD,OAAS,CACZG,QAASD,OAAO,mCAChByB,SAAU9B,6BACVuC,SAAS,GAEVR,qBAAsB7B,KAAMC,SAE7BgC,WAAWC,eAAgB,MACzB"}
```

Comment:
This file shouldn't be changed. Please, revert it.

Reply:
Done, reverted.